### ISSUE TITLE
[BUG] Class name must be a valid object or a string

### ISSUE THREAD

**Issue by shuqingzai** [opcodesio/log-viewer#255]
### Laravel Version

```bash
php artisan --version

Laravel Framework 10.18.0
```

### Package Version

```bash
composer show opcodesio/log-viewer

name     : opcodesio/log-viewer
descrip. : Fast and easy-to-use log viewer for your Laravel application
keywords : arukompas, better-log-viewer, laravel, log viewer, logs, opcodesio
versions : * v3.0.0
type     : library
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
homepage : https://github.com/opcodesio/log-viewer
source   : [git] https://github.com/opcodesio/log-viewer.git 40ed823a38a8b5b175778bc460162ab1d72a8244
dist     : [zip] https://api.github.com/repos/opcodesio/log-viewer/zipball/40ed823a38a8b5b175778bc460162ab1d72a8244 40ed823a38a8b5b175778bc460162ab1d72a8244
```

### BUG

`\Opcodes\LogViewer\Logs\LogType::name()`

https://github.com/opcodesio/log-viewer/blob/40ed823a38a8b5b175778bc460162ab1d72a8244/src/Logs/LogType.php#L48

<img width="747" alt="image" src="https://github.com/opcodesio/log-viewer/assets/44406033/08f29ec8-547f-4308-bd92-0f4e82c4cb54">

**Comment by ahtinurme** [comment]
Second to this.

**Comment by arukompas** [comment]
hey guys, thanks for reporting this! This happened when there were unsupported files found with the glob patterns :) 

This is now fixed in **v3.0.1**. Any files found that are not supported by Log Viewer will be excluded by default. To change the behaviour, set `'hide_unknown_files'` config option to true.

See more here - https://log-viewer.opcodes.io/docs/3.x/configuration/including-excluding-files#hidingshowing-unknown-log-files

Have a great week!
